### ISSUE TITLE
Reminder to add Casey's intro to Escorial Omega 1 12

### ISSUE THREAD

**Issue by caseydue** [homermultitext/hmt-web#37]
This is just a reminder to add the intro that I wrote for Escorial Omega 1 12 to the page for that manuscript (this is the introduction I wrote specifically for this manuscript, not the other article on the Escorial that Mary and I co-wrote and you have already posted - but please link that one to the page as well). Thanks!

**Comment by mebbott** [comment]
Hi Stephanie, This one still needs to be done as well. Let Casey and me know if you don't have the document she is referring to. Thanks!

**Comment by StephanieML** [comment]
Yes, I have it. I'm still waiting on a fly leaf page to get added to the data set so that I can cite the image properly. I will try to get this done as soon as possible.
